MODULE.bazel

```diff
@@ -31,11 +32,33 @@ bazel_dep(name = "rules_python", version = "0.36.0")
 bazel_dep(name = "rules_testing", version = "0.6.0")
 bazel_dep(name = "googletest", version = "1.14.0.bcr.1", repo_name = "com_google_googletest")
 bazel_dep(name = "with_cfg.bzl", version = "0.2.4")
-bazel_dep(name = "abseil-cpp", version = "20230802.1")
+bazel_dep(name = "abseil-cpp", version = "20240116.1")
```

Comment:
Can we update abseil to https://registry.bazel.build/modules/abseil-cpp/20240722.0? This would include https://github.com/abseil/abseil-cpp/commit/65a55c2ba891f6d2492477707f4a2e327a0b40dc

Context: https://github.com/bazelbuild/bazel/commit/193b114287b3e20850a4b106b889771dfa63a601

Reply:
done